tmt/steps/execute/internal.py

```diff
@@ -74,38 +79,41 @@ class ExecuteInternal(tmt.steps.execute.ExecutePlugin):
     # Supported keys
     _keys = ["script", "interactive"]
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args: Any, **kwargs: Any):
         super().__init__(*args, **kwargs)
         self._previous_progress_message = ""
         self.scripts = SCRIPTS
 
     @classmethod
-    def options(cls, how=None):
+    def options(cls, how: Optional[str] = None) -> List[tmt.options.ClickOptionDecoratorType]:
         """ Prepare command line options for given method """
-        options = []
-        # Shell script as a test
-        options.append(click.option(
-            '-s', '--script', metavar='SCRIPT', multiple=True,
-            help='Shell script to be executed as a test.'))
-        # Interactive mode
-        options.append(click.option(
-            '-i', '--interactive', is_flag=True,
-            help='Run in interactive mode, do not capture output.'))
-        # Disable interactive progress bar
-        options.append(click.option(
-            '--no-progress-bar', is_flag=True,
-            help='Disable interactive progress bar showing the current test.'))
-        return options + super().options(how)
-
-    def wake(self, keys=None):
+        options = super().options(how)
+        options[:0] = [
+            click.option(
+                '-s', '--script', metavar='SCRIPT', multiple=True,
+                help='Shell script to be executed as a test.'),
+            # Interactive mode
+            click.option(
+                '-i', '--interactive', is_flag=True,
+                help='Run in interactive mode, do not capture output.'),
+            # Disable interactive progress bar
+            click.option(
+                '--no-progress-bar', is_flag=True,
+                help='Disable interactive progress bar showing the current test.')
+            ]
+        return options
+
+    # TODO: use better types once superclass gains its annotations
+    def wake(self, keys: Optional[List[str]] = None) -> None:
```

Comment:
We got rid of `keys` parameter for all `wake()` methods in https://github.com/teemtee/tmt/pull/1439 because it was unused. Also the `TODO` comment can be deleted?